Rover/AP_Arming.cpp

```diff
@@ -200,6 +200,9 @@ bool AP_Arming_Rover::mode_checks(bool report)
     if (!rover.control_mode->allows_arming()) {
         check_failed(report, "Mode not armable");
         return false;
+    } else if (rover.control_mode == &rover.mode_auto && rover.mode_auto.mission.num_commands() <= 1) {
+        check_failed(report, "No Mission, cannot arm");
```

Comment:
We could leave out the "cannot arm" portion because all arming checks prevent arming.

Alternative wording might be, "Auto requires mission" but "No mission" is also fine.

Reply:
done